### ISSUE TITLE
Handle radiotap and vendor namespace bits in radiotap presence flags

### ISSUE THREAD

**Issue by first-leon** [the-tcpdump-group/tcpdump#498]
See package flying over the wireless interface using tcpdump. I see something like this:
```
23:48:41.505834 24.0 Mb/s 2412 MHz 11g -83dB signal [bit 29] BA RA:60:5b:b4:c7:61:5b (oui Unknown) 
23:48:41.506225 6.0 Mb/s 2412 MHz 11g -83dB signal [bit 29] BA RA:60:5b:b4:c7:61:5b (oui Unknown) 
23:48:41.506544 24.0 Mb/s 2412 MHz 11g -83dB signal [bit 29] BA RA:60:5b:b4:c7:61:5b (oui Unknown) 
23:48:41.557061 24.0 Mb/s 2412 MHz 11g -82dB signal [bit 29] BA RA:60:5b:b4:c7:61:5b (oui Unknown)
```
Each line has a substring «[bit 29]." I can not understand what it means.

Thank You [zuzzas](https://www.linux.org.ru/forum/admin/12250472?lastmod=1452341312133#comment-12253347), he suggested.

"[This bit indicates a field whose size we do not know, so we cannot proceed.  Just print the bit number.](https://github.com/the-tcpdump-group/tcpdump/blob/23d1a76b3ea2e5b12b4e7ea33d829a1679dccdec/print-802_11.c#L2729)"

Maybe you need to add this information in the FAQ(or in MAN)?

```
tcpdump --version
tcpdump version 4.7.4
libpcap version 1.7.4
OpenSSL 1.0.2d 9 Jul 2015
```
```
$ uname -a
Linux notebook 4.2.0-23-generic #28-Ubuntu SMP Sun Dec 27 17:47:31 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by guyharris** [comment]
Or maybe we need to fix tcpdump to handle the [radiotap namespace](http://www.radiotap.org/defined-fields/Radiotap%20Namespace) and [vendor namespace](http://www.radiotap.org/defined-fields/Vendor%20Namespace) bits.

**Comment by guyharris** [comment]
Fixed in ea1182b7f39bd6773b4bff9acc16520f6156004b, which fixes a bunch of issues with radiotap headers.  You should no longer be seeing those indications, as those bits either don't indicate fields at all or indicate a field whose size we *do* know (bit 30), and we *can* and now *do* proceed.  You may also see additional information, such as information for additional radio antennae, now that we handle bit 29.
